dispatches/models/fossil_case/concrete_tube/heat_exchanger_tube.py

```diff
@@ -0,0 +1,503 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Basic IDAES ConcreteTubeSide model Model.
+
+"""
+
+# Import Pyomo libraries
+from pyomo.environ import (
+    SolverFactory,
+    Var,
+    Constraint,
+    units as pyunits
+)
+from pyomo.common.config import ConfigBlock, ConfigValue, In
+
+# Import IDAES cores
+from idaes.core import (
+    ControlVolume1DBlock,
+    UnitModelBlockData,
+    declare_process_block_class,
+    MaterialBalanceType,
+    EnergyBalanceType,
+    MomentumBalanceType,
+    FlowDirection,
+    useDefault,
+)
+from idaes.generic_models.unit_models.heat_exchanger \
+    import HeatExchangerFlowPattern
+from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.util.misc import add_object_reference
+from idaes.core.util.exceptions import ConfigurationError
+from idaes.core.util.tables import create_stream_table_dataframe
+from idaes.core.util.constants import Constants as c
+from idaes.core.util import scaling as iscale
+
+import idaes.logger as idaeslog
+
+
+__author__ = "Konica Mulani, Jaffer Ghouse"
+
+# Set up logger
+_log = idaeslog.getLogger(__name__)
+
+@declare_process_block_class("ConcreteTubeSide")
+class HeatExchanger1DDataNoshell(UnitModelBlockData):
+
+
+    """Standard Heat Exchanger 1D Unit Model Class."""
+
+    CONFIG = UnitModelBlockData.CONFIG()
```

Comment:
Seeing as this model on has one side (i.e. control volume), I don't really see a need for declaring the `SideTemplate` separately. Why not create a single `ConfigBlock` for the unit and make it inherit from the base class like usual (so that you don't need to define "dynamic" and "has_holdup" yourself)?